geocity/apps/submissions/views.py

```diff
@@ -1120,6 +1084,99 @@ def anonymous_submission(request):
             return redirect(step.url)
 
 
+def get_anonymous_submission_by_tags(request, entityfilter, typefilter):
+    # Validate entity
+    entities = AdministrativeEntity.objects.public().filter_by_tags(entityfilter)
+    if len(entities) != 1:
+        raise Http404
```

Comment:
We should inform the integrator that for anonymous submission configuration, tags on administrative entities must be unique, no ?

Reply:
It's done like that because it's unique with `entityfilter` `typefilter`. They don't want to block "urb; pdc; sports; ssp".

And informing is a bad approach, when we could just restrict in the model as unique, and force it to be unique.